packages/react-core/src/components/Tooltip/examples/TooltipBasic.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Tooltip } from '@patternfly/react-core';
+
+export const TooltipBasic: React.FunctionComponent = () => (
+  <div style={{ margin: '100px' }}>
+    <Tooltip
+      content={
+        <div>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam id feugiat augue, nec fringilla turpis.
+        </div>
+      }
+    >
+      <span tabIndex={0} style={{ border: '1px dashed' }}>
+        I have a tooltip!
+      </span>
```

Comment:
It'd be worth updating this `<span>` to a `<Button>` like the other examples to follow the current a11y doc regarding not placing tooltips on static elements (except for truncation).